src/api/endpoints/redemption_requests.py

```diff
@@ -103,60 +123,56 @@ def put(cls, redeem_id=None):
 
     @classmethod
     @token_required
-    @roles_required(["CIO", "President", "Vice President", "Secretary"])
+    @roles_required(["cio", "society president", "vice president", "secretary"])
     def get(cls, redeem_id=None):
         """Get Redemption Requests."""
         if redeem_id:
             redemp_request = RedemptionRequest.query.get(redeem_id)
             return find_item(redemp_request)
         else:
-            search_term_name = request.args.get('name')
+            search_term_name = request.args.get('society')
             if search_term_name:
-                redemp_request = RedemptionRequest.query.filter_by(
+                society = Society.query.filter_by(
                                         name=search_term_name).first()
-                return find_item(redemp_request)
+                if not society:
+                    mes = f"Society with name:{search_term_name} not found"
+                    return {"message": mes}, 400
```

Comment:
We should standardise how we build our responses.
Wasn't this the function of the response builder?